comparative-tests/src/test/java/com/github/bazel_contrib/contrib_rules_jvm/comparative/BUILD.bazel

```diff
@@ -0,0 +1,22 @@
+# gazelle:java_extension disabled
```

Comment:
I'd be tempted to add checked in example output XMLs from each of the test-runners, so someone can inspect them at a glance without having to go through all the hoops of having to run them themselves.

Reply:
Thanks for doing it, it's great to be able to inspect them as I read the fantastic Readme.